spec.md

```diff
@@ -3776,6 +3830,7 @@ statement =
   | force_release , [ info ]
   | "connect" , reference , "," , expr , [ info ]
   | "invalidate" , reference , [ info ] ;
+  | "propassign" , reference , "," , reference , [ info ]
```

Comment:
Should these be static references? (and if so probably should have some language about this)

Reply:
Yeah I think static reference is appropriate for the lhs. I am realizing that we will want to allow property literals in the rhs in the future, but those haven't been specified yet.

Where do you think it would make sense to have some language about this? I actually just removed a part of the new section that was trying to talk about the types of expressions allowed in the propassign statement, because we already document the places property types can be used in a separate section: https://github.com/chipsalliance/firrtl-spec/blob/main/spec.md#property-types